File: Bedu/DesarrolloMobileJava/Sesion2/RetoSesion2/app/src/main/java/xyz/eliothmonroy/retosesion2/MainActivity.java
package xyz.eliothmonroy.retosesion2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    Button enviar;
    EditText edadText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        enviar=findViewById(R.id.enviar);
        edadText=findViewById(R.id.edad);

        enviar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent;
                if (!edadText.getText().toString().equals("")){
                    int edad=Integer.parseInt(edadText.getText().toString());
                    if (edad>17){
                        intent=new Intent(getApplicationContext(),MayorEdadActivity.class);
                    }else{
                        intent=new Intent(getApplicationContext(),MenorEdadActivity.class);
                    }
                    startActivity(intent);
                }else{
                    Toast.makeText(getApplicationContext(),"Por favor ingrese su edad", Toast.LENGTH_LONG).show();
                }

            }
        });

    }
}

File: Bedu/KotlinIntermedio/Sesion4/Sesion4/app/src/main/java/xyz/eliothmonroy/sesion4/MainActivity.kt
package xyz.eliothmonroy.sesion4

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button.setOnClickListener {
            obtenerTexto()
        }

    }

    private fun obtenerTexto() {
        if(editText.text.isNotEmpty()){
            editText.visibility= View.GONE
            textView.text=editText.text.toString()
        }else{
            Toast.makeText(this,"Por favor ingrese un valor", Toast.LENGTH_SHORT).show()
        }
    }
}
